experimental/framework/fluid-static/README.md

```diff
@@ -45,20 +45,20 @@ const container = await Fluid.getContainer("_unique-id_", /* config */);
 
 ## Defining Fluid Containers
 
-Fluid Containers are defined by a config. The config includes initial properties of the Container as well as what collaborative objects can be dynamically created.
+Fluid Containers are defined by a schema. The schema includes initial properties of the Container as well as what collaborative objects can be dynamically created.
 
-See [`ContainerConfig`](./src/types.ts) in [`./src/types/ts`](./src/types.ts) for details about the specific properties.
+See [`ContainerSchema`](./src/types.ts) in [`./src/types/ts`](./src/types.ts) for details about the specific properties.
 
 ```javascript
-const config = {
+const schema = {
     name: "my-container",
     initialObjects: {
         /* ... */
     },
     dynamicObjectTypes: [ /*...*/ ],
 }
-await Fluid.createContainer("_unique-id_", config);
-const container = await Fluid.getContainer("_unique-id_", config);
+await Fluid.createContainer("_unique-id_", schema);
```

Comment:
This makes more sense if it's in the TinyliciousClient

```suggestion
await TinyliciousClient.createContainer("_unique-id_", schema);
```

Reply:
Looks like you already moved it to TinyliciousClient. We can probably just remove it from here :)

Reply:
Yep, removed from here and link added to the TinyliciousClient doc